snow/consensus/snowman/poll/early_term_traversal.go

```diff
@@ -229,27 +231,17 @@ func (p *earlyTermPoll) Finished() bool {
 	// We should terminate the poll only when votes for all IDs or prefixes cannot be improved.
 	if weCantImproveVoteForSomeIDOrPrefix {
 		p.finished = true
+		p.metrics.observeEarlyAlpha(time.Since(p.start))
```

Comment:
For all the other observations for this metric, we do it once per poll. But we called `shouldTerminateDueToConfidence` (changed to `shouldTerminate`) multiple times per poll. So this should be moved out to here.